docs/management/commands/update_docs.py

```diff
@@ -175,29 +177,32 @@ def build_doc_release(self, release, force=False):
 
             if self.verbosity >= 2:
                 self.stdout.write(f"  building {builder} ({source_dir} -> {build_dir})")
-            try:
+            sphinx_init = {
+                "srcdir": source_dir,
+                "confdir": source_dir,
+                "outdir": build_dir,
+                "doctreedir": build_dir.joinpath(".doctrees"),
+                "buildername": builder,
```

Comment:
I just realized that the `-j auto` from the previous version was not carried over. I think this should be equivalent:
```suggestion
                "buildername": builder,
                "parallel": multiprocessing.cpu_count(),
```